swagger/v1/openapi.json

```diff
@@ -65,34 +59,6 @@
         "responses": {
           "200": {
             "description": "lists all benchmarks requested",
-            "schema": {
-              "type": "object",
-              "properties": {
-                "meta": {
-                  "$ref": "#/definitions/metadata"
-                },
-                "links": {
-                  "$ref": "#/definitions/links"
-                },
-                "data": {
-                  "type": "array",
-                  "items": {
-                    "properties": {
-                      "type": {
-                        "type": "string"
-                      },
-                      "id": {
-                        "type": "string",
-                        "format": "uuid"
-                      },
-                      "attributes": {
-                        "$ref": "#/definitions/benchmark"
-                      }
-                    }
-                  }
-                }
-              }
-            },
             "examples": {
```

Comment:
This is still a problem, unfortunately. Response `"examples"` should be nested under `content => application/json` https://swagger.io/specification/#example-object

```yaml
responses:
  '200':
    description: your car appointment has been booked
    content: 
      application/json:
        schema:
          $ref: '#/components/schemas/SuccessResponse'
        examples:
          confirmation-success:
            $ref: '#/components/examples/confirmation-success'
```